bin/bugsnag-maze-runner

```diff
@@ -75,12 +122,51 @@ class MazeRunnerEntry
     @args = args.dup
     options = @parser.parse args
 
+    # Process CL options
     init if options[:init]
-    MazeRunner.configuration.appium_session_isolation = true if options[OPTION_SEPARATE_SESSIONS]
+    MazeRunner.configuration.appium_session_isolation = options[OPTION_SEPARATE_SESSIONS]
+    MazeRunner.configuration.app_location = options[OPTION_APP]
+    farm = options[OPTION_FARM]
+    device_type = MazeRunner.configuration.device_type = options[OPTION_DEVICE]
+    MazeRunner.configuration.farm = if farm.nil?
+                                      :none
+                                    elsif farm == 'bs'
+                                      :bs
+                                    elsif farm == 'local'
+                                      :local
+                                    else
+                                      raise "Unknown farm '#{farm}'"
+                                    end
```

Comment:
Indenting the whole block instead of just the first line seems unusual style-wise. Could probably use a ruby second opinion. @imjoehaines ?

Some of the instances where many `elseif` statements are used could be made more compact with `case`/`when`

```suggestion
    MazeRunner.configuration.farm = case farm
    when nil then :none
	when 'bs' then :bs
	when 'local' then :local
	else
	  raise "Unknown farm '#{farm}'"
	end
```

Reply:
It's a recent-ish Rubocop rule. I'm not a fan personally but I think it's on by default

Reply:
ah, I wondered if it was something like that. thanks!

Reply:
Yeah, I just blindly followed what Rubocop told me to do on this one.  I imagine we'll have a discussion about landing on some kind of Rubocop config at some point and stickling to it (there are still plenty of violations at the moment that I'm gradually working through).

Reply:
Oh, I have changed it to a `case` now though, much better.